samples/excel/85-preview-apis/range-merged-ranges.yaml

```diff
@@ -1,34 +1,52 @@
 order: 7
-id: excel-merged-areas
-name: Merged areas
-description: This sample shows how to create and then find the merged areas in a worksheet.
+id: excel-merged-ranges
+name: Merged ranges
+description: This sample shows how to create and find merged ranges in a worksheet.
 host: EXCEL
 api_set:
     ExcelAPI: '1.13'
 script:
     content: |
       $("#setup").click(() => tryCatch(setup));
-      $("#get-merged-areas").click(() => tryCatch(getMergedAreas));
+      $("#create-merged-range").click(() => tryCatch(createMergedRange));
+      $("#get-merged-range").click(() => tryCatch(getMergedRange));
 
-      async function getMergedAreas() {
+      async function createMergedRange() {
+        await Excel.run(async (context) => {
+          // Retrieve the worksheet and the table in that worksheet.
+          const sheet = context.workbook.worksheets.getActiveWorksheet();
+          const range = sheet.getRange("B2:E6");
+
+          // Create a merged range in the first row of the table.
+          const chartTitle = range.getRow(0);
+          chartTitle.merge(true);
+
+          // Format the merged range.
+          chartTitle.format.horizontalAlignment = "Center";
+
+          await context.sync();
+        });
+      }
+
+      async function getMergedRange() {
         await Excel.run(async (context) => {
           // Retrieve the worksheet and the table in that worksheet.
           const sheet = context.workbook.worksheets.getActiveWorksheet();
           const table = sheet.getRange("B2:E6");
```

Comment:
Nit: it's a little confusing that this range is named `table`. Consider renaming it something like `tableRange` here and elsewhere in the sample.

Reply:
This is very helpful, thank you! I wasn't sure what to call it.